mets_adapter/src/main/scala/uk/ac/wellcome/mets_adapter/IngestUpdate.scala

```diff
@@ -0,0 +1,18 @@
+package uk.ac.wellcome.mets_adapter.models
+
+case class IngestUpdate(space: Space, bag: StubBag)
+
+case class Space(id: String)
+sealed trait BaseBag {
+  val info: BagInfo
+}
+case class StubBag(info: BagInfo) extends BaseBag
+case class Bag(info: BagInfo, manifest: BagManifest, location: BagLocation)
+    extends BaseBag
+case class BagInfo(externalIdentifier: String)
+
+case class BagManifest(files: List[BagFile])
+
+case class BagLocation(bucket: String, path: String)
+
+case class BagFile(name: String, path: String)
```

Comment:
Should these be shared between the adapter and transformer?

Reply:
I don't think the transformer should know anything about Bags, it should only understand METS XML